compose-builder/Makefile

```
@@ -19,7 +19,7 @@
 include .env
 
 COMPOSE_FILES:=-f docker-compose-base.yml
-OPTIONS:=" arm64 no-secty mqtt dev app-dev taf-secty taf-no-secty taf-perf taf-perf-no-secty ui ds-bacnet ds-camera ds-grove ds-modbus ds-mqtt ds-random ds-rest ds-snmp ds-virtual modbus-sim asc-http asc-http-s asc-mqtt asc-mqtt-s " # Must have spaces around words for `filter-out` function to work properly
+OPTIONS:=" arm64 no-secty mqtt-bus mqtt-broker dev app-dev taf-secty taf-no-secty taf-perf taf-perf-no-secty ui ds-bacnet ds-camera ds-grove ds-modbus ds-mqtt ds-random ds-rest ds-snmp ds-virtual modbus-sim asc-http asc-http-s asc-mqtt asc-mqtt-s " # Must have spaces around words for `filter-out` function to work properly
```

```suggestion
 # Must have spaces around words for `filter-out` function to work properly
define OPTIONS
arm64 no-secty mqtt dev app-dev taf-secty taf-no-secty taf-perf taf-perf-no-secty ui 
 ds-bacnet ds-camera ds-grove ds-modbus ds-mqtt ds-random ds-rest ds-snmp ds-virtual 
 modbus-sim asc-http asc-http-s asc-mqtt asc-mqtt-s
endef
export OPTIONS

```
This is getting too lengthy in a long-line and hard to read, 
how about change this to multi-lines using "define"? 
so, it would be like this:
```make
 # Must have spaces around words for `filter-out` function to work properly
define OPTIONS
arm64 no-secty mqtt dev app-dev taf-secty taf-no-secty taf-perf taf-perf-no-secty ui 
 ds-bacnet ds-camera ds-grove ds-modbus ds-mqtt ds-random ds-rest ds-snmp ds-virtual 
 modbus-sim asc-http asc-http-s asc-mqtt asc-mqtt-s
endef
export OPTIONS
```

and when we want to evaluate the value, use `$$OPTIONS`.  eg. in line 181 (original line number), change 
`SERVICES:=$(filter-out $(OPTIONS),$(ARGS))` to 
`SERVICES:=$(filter-out "$$OPTIONS", $(ARGS))`